lib/shopify_app/access_scopes/shop_strategy.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ShopifyApp
+  module AccessScopes
+    class ShopStrategy
+      class << self
+        def update_access_scopes?(shop_domain)
+          shop_access_scopes = shop_access_tokens(shop_domain)
+          return true unless shop_access_scopes
+          configuration_access_scopes != shop_access_tokens(shop_domain)
```

Comment:
Why are you calling `shop_access_tokens` again?

Reply:
Fixed to `offline_access_tokens`